src/index.ts

```diff
@@ -516,9 +516,10 @@ function setMeteosatImages(latest: {date: Date, image: string}, imageType: Image
       // copy canvas into output in one step
       const output = document.getElementById("output") as HTMLCanvasElement;
       const outCtx = output.getContext("2d");
-      outCtx.canvas.width = METEOSAT_WIDTH;
-      outCtx.canvas.height = METEOSAT_WIDTH;
-      outCtx.drawImage(canvas, 0, 0);
+      // Feels a bit like a cludge
+      outCtx.canvas.width = window.innerWidth;
```

Comment:
We usually do this kind of resizing in CSS. You want the canvas to be the size of the image data.

Reply:
But if I set the canvas to the data's size, it displays zoomed in to 100%.